### ISSUE TITLE
BukkitUserVanishEvent error

### ISSUE THREAD

**Issue by JuraRusan** [Syrent/SayanVanish#333]
When logging into the server, an error occurs.
And because of this a warning appeared with `velocitab`

paper [console.log](https://paste.helpch.at/xicipijudi.log)
velocity [console.log](https://paste.helpch.at/eqidelepob.log)

One time I didn’t have an error, everything worked perfectly, and the player disappeared in the table, but I can’t understand how this happened, I tested for 4 hours, but I still didn’t understand. I was never able to reproduce it without errors again.

If necessary, I can provide settings from the servers.
Thank you!

**Comment by JuraRusan** [comment]
But if something is due to the effects, then I changed almost nothing in them, I just turned it on.

```yaml
effects:
-   use-packet: true
    keep-after-appear: false
    type: NIGHT_VISION
    duration: 2147483647
    amplifier: 0
    ambient: false
    particles: false
-   use-packet: true
    keep-after-appear: false
    type: WATER_BREATHING
    duration: 2147483647
    amplifier: 0
    ambient: false
    particles: false
-   use-packet: true
    keep-after-appear: false
    type: FIRE_RESISTANCE
    duration: 2147483647
    amplifier: 0
    ambient: false
    particles: false
id: effect
enabled: true
```

**Comment by Syrent** [comment]
fixed in 1.5.0-SNAPSHOT versions
